mmaction/models/recognizers/recognizer2d.py

```diff
@@ -6,7 +6,7 @@
 class Recognizer2D(BaseRecognizer):
     """2D recognizer model framework."""
 
-    def forward_train(self, imgs, labels):
+    def forward_train(self, imgs, labels, aux_info):
```

Comment:
can this `aux_info` thing be replaced with `**kwargs`? since there might be more additional keywargs for users to add.